File: CS/SupabaseApiCall/SupabaseApiCall/Models/FatturaCliente.cs
using Supabase.Postgrest.Attributes;
using Supabase.Postgrest.Models;


namespace SupabaseApiCall.Models
{
    [Table("VFATTURECLIENTI")]
    public class FatturaCliente : BaseModel
    {
        [Column("CodiceCliente")]
        public long CodiceCliente { get; set; }
        [Column("DataFattura")]
        public DateTime DataFattura { get; set; }
        [Column("NumeroFattura")]
        public long NumeroFattura { get; set; }
        [Column("CodiceArticolo")]
        public String? CodiceArticolo { get; set; }      //Nullable string  
        [Column("QtaMovimento")]
        public double QtaMovimento { get; set; }
        [Column("Prezzo")]
        public double Prezzo { get; set; }
        [Column("ScontoPerc")]
        public double ScontoPerc { get; set; }
        [Column("Importo")]
        public double Importo { get; set; }
        [Column("ImportoIvato")]
        public double ImportoIvato { get; set; }
        [Column("ImportoIva")]
        public double ImportoIva { get; set; }
        [Column("AliquotaIva")]
        public int AliquotaIva { get; set; }
    }
}

File: Go/Sincronize_Data/models/FatturaCliente.go
package models

import (
	"database/sql"
	"time"
)

type FatturaCliente struct {
	documento
}

func (fc FatturaCliente) GetCodiceControparte() int64 {
	return fc.CodiceControparte
}
func (fc FatturaCliente) GetDataDocumento() time.Time {
	return fc.DataDocumento
}
func (fc FatturaCliente) GetNumeroDocumento() int64 {
	return fc.NumeroDocumento
}

func (fc FatturaCliente) GetCodiceArticolo() string {
	return fc.CodiceArticolo
}
func (fc FatturaCliente) GetQtaMovimento() float64 {
	return fc.QtaMovimento
}
func (fc FatturaCliente) GetPrezzo() float64 {
	return fc.Prezzo
}
func (fc FatturaCliente) GetScontoPerc() float64 {
	return fc.ScontoPerc
}
func (fc FatturaCliente) GetImporto() float64 {
	return fc.Importo
}
func (fc FatturaCliente) GetImportoIvato() float64 {
	return fc.ImportoIvato
}
func (fc FatturaCliente) GetImportoIva() float64 {
	return fc.ImportoIva
}
func (fc FatturaCliente) GetAliquotaIva() sql.NullFloat64 {
	return fc.AliquotaIva
}
